pkg/expr/reader.go

```diff
@@ -0,0 +1,154 @@
+package expr
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data/utils/jsoniter"
+
+	"github.com/grafana/grafana/pkg/expr/classic"
+	"github.com/grafana/grafana/pkg/expr/mathexp"
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
+)
+
+type ExpressionQuery struct {
```

Comment:
Since we have `expr.Query` at https://github.com/grafana/grafana/blob/fabaff9a248f0f31ccdb74c7ca5394286e8f3521/pkg/expr/transform.go#L26-L34 , can this get a comment on what it is, clarifying the difference

Reply:
yes -- i'm hoping it will replace that structure